e2e/command_test.go

```diff
@@ -722,6 +722,92 @@ task {
 		"event count did not increment once. task was not triggered as expected")
 }
 
+// TestE2E_RecreateBadTask tests that after attempting to create a bad task, cleanup is performed
+// so that a subsequent creation of the same task is permissible
+func TestE2E_RecreateBadTask(t *testing.T) {
+	setParallelism(t)
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "recreate_bad_task_cmd")
+
+	srv := newTestConsulServer(t)
+	defer srv.Stop()
+	cts := ctsSetup(t, srv, tempDir, dbTask())
+
+	var taskConfig hclConfig
+	taskName := "new-task"
+	runs := []struct {
+		inputTask     string
+		isExpectExist bool
+	}{
+		{
+			inputTask: fmt.Sprintf(`
+task {
+  name           = "%s"
+  description    = "Creates a new task"
+  module         = "./no-exist"
+  providers      = ["local"]
+  services       = ["web"]
+  enabled = true
+}`, taskName),
+			isExpectExist: false,
+		}, {
+			inputTask: fmt.Sprintf(`
+task {
+  name           = "%s"
+  description    = "Creates a new task"
+  module         = "./test_modules/local_instances_file"
+  providers      = ["local"]
+  services       = ["web"]
+  enabled = true
+}`, taskName),
+			isExpectExist: false,
```

Comment:
Shouldn't this be true? This would be the task that is created successfully after the first bad one failed, right?

Reply:
yep, thanks for catching